jest.setup.js

```diff
@@ -1 +1,2 @@
-import '@testing-library/jest-dom/extend-expect';
+import "@testing-library/jest-dom/extend-expect";
+import "whatwg-fetch";
```

Comment:
Why do we need whatwg-fetch here? I don't see any mock API in the test files yet or I missed it?

Reply:
deleted